inst/tests/tests.Rraw

```diff
@@ -21141,3 +21141,13 @@ dt = data.table(id = 1:25)
 test(2314.1, any(grepl("<int>", tail(capture.output(print(dt, class = TRUE)), 2))), TRUE)
 # Test that class=TRUE with col.names="top" doesn't show classes at bottom
 test(2314.2, !any(grepl("<int>", tail(capture.output(print(dt, class = TRUE, col.names = "top")), 2))), TRUE)
+
+# Test that tzone behavior stops warning with ignore_tzone=TRUE
+tm1 = .POSIXct(1, "UTC")
+dt1 = data.table(a = .POSIXct(0), b = .POSIXct(2, "UTC"))
+test(2315.1, dt1[, between(tm1, a, b)], error = "different tzone attributes")
+test(2315.2, dt1[, between(tm1, a, b, ignore_tzone=TRUE)])
+tm2 <- .POSIXct(1)
+dt2 = data.table(a = .POSIXct(0, "UTC"), b = .POSIXct(2, "UTC"))
+test(2315.3, dt2[, between(tm2, a, b)], message = "mismatched tzone attributes")
+test(2315.4, dt2[, between(tm2, a, b)])
```

Comment:
what's the difference vs. 2315.3?

Reply:
typo
(side note: I cannot check/compile on mac, it's failing with OMP missing symbols)